src/Gesture.ts

```diff
@@ -151,3 +152,18 @@ export const preserveMultiplicativeOffset = (
     offset
   ]);
 };
+
+export const onScroll = (contentOffset: {
+  x?: Animated.Node<number>;
+  y?: Animated.Node<number>;
+}) =>
+  event(
+    [
+      {
+        nativeEvent: {
+          contentOffset
+        }
+      }
+    ],
+    { useNativeDriver: true }
```

Comment:
I'm pretty sure that you don't need `useNativeDriver` for Reanimated event

Reply:
true :)